lib/components/message-ai/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { AIModelName, ChatMessageAI } from "@appflowy-chat/types";
+import { FC } from "react";
+import MessageResponseContent from "../message-response-content";
+import AvatarAI from "../avatar-ai";
+import ActionBarAI from "../action-bar-ai";
+import {
+  ActionBarAIButtonsHoverRes,
+  ActionBarAIButtonsLastRes,
+} from "@appflowy-chat/utils/actionBarAIButtons";
+import "./index.css";
+
+interface IProp {
+  message: ChatMessageAI;
+  isGenerating?: boolean;
+  isLastResponse: boolean;
+  onAIModelChange: (option: AIModelName) => void;
+}
+
+const MessageAI: FC<IProp> = ({ message, isLastResponse, onAIModelChange }) => {
+  function handleOnAIModelChange(option: AIModelName) {
+    console.log(option, "option");
```

Comment:
How can I disable log on production when using console.log

Reply:
Added the Vite plugin that removes logs during the build https://www.npmjs.com/package/vite-plugin-remove-console